src/common/components/AccountingCode/AccountingCodeValue.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+
+import {
+  KeyValue,
+  NoValue,
+} from '@folio/stripes/components';
+
+const AccountingCodeValue = ({ labelId, value }) => (
+  <KeyValue
+    label={<FormattedMessage id={labelId} />}
+    value={value || <NoValue />}
+  />
+);
+
+AccountingCodeValue.propTypes = {
+  labelId: PropTypes.string.isRequired,
```

Comment:
Is there a possibility that displaying a value of AccountingCode field could have different labels?